.github/README.md

```diff
@@ -143,7 +143,9 @@ If that doesn't work, see the [detailed installation instructions](/docs/install
 
 <details>
 <summary>From Source Code</summary>
+  
```

Comment:
From what I can see, this doesn't change the rendered document. Is the newline needed?

Reply:
When playing with the Github preview it seemed necessary to get `[here](/docs/installation/source.md)` to render as a link. Currently, that render is not working. And, looking at it some more, it's also needed for the `* [Rubygem]` and Homebrew shield to render properly, both are also currently not rendering right. I'll fix those too.